### DISCUSSION TITLE
Looking for a contractor to help us

### DISCUSSION THREAD

**Discussion by patrickelectric** [sphinx-doc/sphinx#12891] [General]
We are looking to find someone to create a theme for our software libraries and project, not sure where to look for someone :)

**Comment by ppiegaze** [comment]
For what it's worth, my firm is also looking for someone with experience building Sphinx themes. We are also having some difficulty finding people with this particular skill-set. Any advice would be appreciated.

**Comment by stevepiercy** [comment]
[Sphinx Theme Builder](https://sphinx-theme-builder.readthedocs.io/en/latest/) is the tool to use.

I use it to develop a custom theme [plone-sphinx-theme](https://plone-sphinx-theme.readthedocs.io/) which is based on [sphinx-book-theme](https://sphinx-book-theme.readthedocs.io/en/latest/) which is based on [pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/). I think all these themes use STB.

**Comment by atrnh** [comment]
@patrickelectric @ppiegaze Are you both still looking for a contractor? You can check out some of the Sphinx themes and extensions I've authored/maintained [over here](https://github.com/orgs/SEIBootcamps/repositories?q=sphinx). I'll admit that most (if not all) of my Sphinx repos are lacking in documentation and screenshots, but feel free to reach out and I can provide more concrete examples. You'll find my email in my GitHub profile!

**Comment by liborjelinek** [comment]
A bit late to the party. I am crafting Sphinx Themes almost on a daily basis. I am the Sphinx extension and themes author. For example, the recent public custom themes are https://docs.frontstage.online,  https://documatt.com, https://documatt.com/sphinx-reredirects/. Will be happy to hear more about your expectations.

**Comment by aylosenki** [comment]
cellular automata, symbolic referencing, digital minerals, geometry. = Infinite scroll.

**Comment by dgarcia360** [comment]
We created the theme for @patrickelectric software libraries!

At [TechDocs Studio](https://techdocs.studio/), we have developed Sphinx themes for a variety of projects over the years:

- https://techdocs.studio/work/coiled-docs
- https://techdocs.studio/work/voxel51
- https://techdocs.studio/work/scylladb
- https://techdocs.studio/work/bluerobotics

We have also built extensions, including a [chatbot for docs](https://docs.biel.ai/installation/sphinx), a [feedback widget for docs](https://docs.pushfeedback.com/installation/sphinx), and automation solutions for our clients.  

If you need custom Sphinx themes, extensions, automated documentation from code, or integrations, [let's connect](https://techdocs.studio/contact).
